src/libs/localStore/storage.ts

```diff
@@ -100,11 +100,16 @@ export const getData = async <T>(name: StorageKey): Promise<T | null> => {
 
 export const clearCozyData = async (): Promise<void> => {
   try {
-    const keys = Object.values(CozyPersistedStorageKeys)
+    const keys = storage.getAllKeys()
+    const keysToKeep = Object.values(
+      DevicePersistedStorageKeys
+    ) as unknown as string
```

Comment:
I think we will also remove UserPersistedStorageKeys here no? Which we should not.